paramiko/sftp_file.py

```diff
@@ -473,9 +477,9 @@ def prefetch(self, file_size=None):
             chunks.append((n, chunk))
             n += chunk
         if len(chunks) > 0:
-            self._start_prefetch(chunks)
+            self._start_prefetch(chunks, max_outstanding_requests)
 
-    def readv(self, chunks):
+    def readv(self, chunks, max_outstanding_requests = None):
```

Comment:
It looks like you haven't added any use of `max_outstanding_requests` in `readv()` here -- seems like either it shouldn't be added, or there's a piece of implementation missing.

Reply:
Good catch! I meant to forward the parameter to `_start_prefetch`.